<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:disabled{ 
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 gray, stop:1 white); 	
 }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Image</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="labelImage">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>352</width>
              <height>288</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Image From Camera</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_1">
         <attribute name="title">
          <string>Settings</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10" stretch="1,1">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Face Recognition:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_8">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
                       <item>
                        <widget class="QLabel" name="label_4">
                         <property name="text">
                          <string>Actual Person:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QComboBox" name="comboBoxAvailablePeople">
                         <property name="styleSheet">
                          <string notr="true">QComboBox {
   color: black;
   border-color: blue;
	border-radius: 2px ;
	border-width: 2px;
}
QComboBox:hover{
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 lightgreen, stop:1 white);
 }</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_9">
                       <item>
                        <widget class="QLabel" name="label_9">
                         <property name="styleSheet">
                          <string notr="true"/>
                         </property>
                         <property name="text">
                          <string>TextLabel</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEditNewPerson">
                         <property name="styleSheet">
                          <string notr="true">QLineEdit {
   color: black;
   border-color: blue;
	border-radius: 2px ;
	border-width: 2px;	
}
QLineEdit:hover{
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 lightgreen, stop:1 white);
 }</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_8">
                       <item>
                        <widget class="QPushButton" name="pushButtonAddPerson">
                         <property name="styleSheet">
                          <string notr="true">QPushButton {
   color: black;
	border-radius: 2px;
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 lightblue, stop:1 white); 		
}
QPushButton:hover{
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 yellow, stop:1 white);
 }</string>
                         </property>
                         <property name="text">
                          <string>Add Person</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="pushButtonRemovePerson">
                         <property name="styleSheet">
                          <string notr="true">QPushButton {
   color: black;
	border-radius: 2px;
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 lightblue, stop:1 white); 		
}
QPushButton:hover{
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 red, stop:1 white);
 }</string>
                         </property>
                         <property name="text">
                          <string>Remove Person</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButtonTrain">
                   <property name="styleSheet">
                    <string notr="true">QPushButton {
   color: black;
	border-radius: 2px;
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 lightblue, stop:1 white); 		
}
QPushButton:hover{
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 blue, stop:1 white);
 }</string>
                   </property>
                   <property name="text">
                    <string>Train</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Camera:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_2">
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
                     <item>
                      <spacer name="horizontalSpacer_4">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButtonMoveUp">
                       <property name="enabled">
                        <bool>false</bool>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton {
   color: black;
	border-radius: 2px;
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 lightblue, stop:1 white); 		
}
QPushButton:hover{
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 yellow, stop:1 white);
 }
QPushButton:disabled{ 
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 gray, stop:1 white); 	
 }</string>
                       </property>
                       <property name="text">
                        <string>Up</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_5">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
                     <item>
                      <widget class="QPushButton" name="pushButtonMoveLeft">
                       <property name="enabled">
                        <bool>false</bool>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton {
   color: black;
	border-radius: 2px;
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 lightblue, stop:1 white); 		
}
QPushButton:hover{
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 yellow, stop:1 white);
 }
QPushButton:disabled{ 
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 gray, stop:1 white); 	
 }</string>
                       </property>
                       <property name="text">
                        <string>Left</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButtonMoveDown">
                       <property name="enabled">
                        <bool>false</bool>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton {
   color: black;
	border-radius: 2px;
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 lightblue, stop:1 white); 		
}
QPushButton:hover{
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 yellow, stop:1 white);
 }
QPushButton:disabled{ 
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 gray, stop:1 white); 	
 }</string>
                       </property>
                       <property name="text">
                        <string>Down</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButtonMoveRight">
                       <property name="enabled">
                        <bool>false</bool>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton {
   color: black;
	border-radius: 2px;
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 lightblue, stop:1 white); 		
}
QPushButton:hover{
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 yellow, stop:1 white);
 }
QPushButton:disabled{ 
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 gray, stop:1 white); 	
 }</string>
                       </property>
                       <property name="text">
                        <string>Right</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBoxEnableMovement">
                   <property name="text">
                    <string>Enable Movement</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Pasek Boczny</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStart">
          <property name="styleSheet">
           <string notr="true">QPushButton {
   color: black;
	border-radius: 2px;
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 lightblue, stop:1 white); 		
}
QPushButton:hover{
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 green, stop:1 white);
 }
QPushButton:disabled{ 
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 gray, stop:1 white); 	
 }</string>
          </property>
          <property name="text">
           <string>Start </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
   color: black;
	border-radius: 2px;
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 lightblue, stop:1 white); 		
}
QPushButton:hover{
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 red, stop:1 white);
 }

QPushButton:disabled{ 
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 gray, stop:1 white); 	
 }</string>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonGetImage">
          <property name="styleSheet">
           <string notr="true">QPushButton {
   color: black;
	border-radius: 2px;
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 lightblue, stop:1 white); 		
}
QPushButton:hover{
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 yellow, stop:1 white);
 }

QPushButton:disabled{ 
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 gray, stop:1 white); 	
 }</string>
          </property>
          <property name="text">
           <string>Get Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAddFace">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
   color: black;
	border-radius: 2px;
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 lightblue, stop:1 white); 		
}
QPushButton:hover{
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 yellow, stop:1 white);
 }

QPushButton:disabled{ 
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 gray, stop:1 white); 	
 }</string>
          </property>
          <property name="text">
           <string>Add Face</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,0,1,1,0,1,1">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Mode:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelMode">
          <property name="text">
           <string>Stoped</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Actual Person:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelActualPerson">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Frames Captured</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFramesCaptured">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="LedIndicator" name="ledIndicatorWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">QPushButton:disabled{ 
	background: qlineargradient(x1:1, y1:0, x2:1, y2:1,
	stop:0 white, stop: 0.8 gray, stop:1 white); 	
 }</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LedIndicator</class>
   <extends>QWidget</extends>
   <header>ledindicator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
